packages/react/package.json

```diff
@@ -47,7 +47,7 @@
     "react": "^18.2.0",
     "react-i18next": "^13.3.1",
     "tailwind-merge": "^2.0.0",
-    "thirdweb": "^5.28.0",
+    "thirdweb": "^5.29.1",
```

Comment:
Not really needed, as the ^ in front means the package will be updated to 5.x. Since there were no changes to `package-lock.json`, it means both packages were on the same version anyway. That said, there's no harm updating these either, so the change is fine.

Reply:
Hadn't run `npm i`, after doing so the package.lock had changed slightly

Reply:
Yeah, mainly just a copy of what's in `package.json`. You can find the actual version being used here: https://github.com/TreasureProject/tdk-js/blob/main/package-lock.json#L27336